designsafe/apps/api/tasks.py

```diff
@@ -885,3 +884,86 @@ def email_user_publication_request_confirmation(self, username):
             [user.email],
             html_message=email_body
         )
+
```

Comment:
We should add the shared task decorator to this task so that it will run on the default queue. Something like: `@shared_task(bind=True, max_retries=1, default_retry_delay=60)`

Reply:
Added